csrc/ir/nodes.cpp

```diff
@@ -2103,7 +2109,7 @@ std::vector<PolymorphicValue> ExpandOp::evaluate(
   for (auto i : c10::irange(1, inputs.size())) {
     expanded_size.push_back((int64_t)inputs.at(i));
   }
-  return {at::expand_copy(in, expanded_size)};
+  return {in.expand(expanded_size)};
```

Comment:
I believe this change is warranted regardless of whether we go forward with squeezing expanded ops. It was necessary in this case to fix broken test `ExpandReduce_CUDA`. cc @Priya2698